crates/ty/src/args.rs

```diff
@@ -51,14 +51,19 @@ pub(crate) struct CheckCommand {
     #[arg(long, value_name = "PROJECT")]
     pub(crate) project: Option<SystemPathBuf>,
 
-    /// Path to the Python installation from which ty resolves type information and third-party dependencies.
+    /// Path to the Python environment from which ty resolves type information and third-party
+    /// dependencies.
     ///
-    /// If not specified, ty will look at the `VIRTUAL_ENV` environment variable.
+    /// If not specified, ty will attempt to infer it from the `VIRTUAL_ENV` environment variable or
+    /// discover a `.venv` directory in the project root or working directory.
     ///
-    /// ty will search in the path's `site-packages` directories for type information and
-    /// third-party imports.
+    /// If a path to a Python interpreter is provided, e.g., `.venv/bin/python3`, ty will attempt to
+    /// find an environment two directories up from the interpreter's path, e.g., `.venv`. At this
+    /// time, ty does not invoke the interpreter to determine the location of the environment. This
+    /// means that ty will not resolve dynamic executables such as a shim.
     ///
-    /// This option is commonly used to specify the path to a virtual environment.
+    /// ty will search in the given path's `site-packages` directories for type information and
```

Comment:
maybe

```suggestion
    /// ty will search in the resolved installation's `site-packages` directories for type information and
```

?

Reply:
I don't want to use "installation" because it includes virtual environments, but I can use "resolved environment's"

Reply:
that works for me!